help-docs/marketplace/apps/proof-of-humanity.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 11
+title: Proof of Humanity
+---
+
+### About this app
+
+Rarimo’s Proof-of-Humanity plug-in ensures that only verified humans are present either within a server or channel. It keeps the community server spam and scam-free, which saves mods’ time, improves the quality of conversation, and helps protect airdrops.
+
+The solution is uniquely robust yet does not require users to reveal any personal details. Users can select between five identity providers to verify their humanity and will receive a verified human credential that they can use across Web3 for perks and exclusive access.

```

Comment:
looks like the flow involves an NFT on polygon for PoH access. please describe a bit about it here to clarify for users. a direct link to your docs on the topic would be helpful also